samples/secure-fraud-detection/src/main/java/io/quarkiverse/langchain4j/sample/Setup.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkiverse.langchain4j.sample;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+
+import io.quarkus.runtime.StartupEvent;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.transaction.Transactional;
+
+@ApplicationScoped
+public class Setup {
+
+    public static List<String> CITIES = List.of("Paris", "Lyon", "Marseille", "Bordeaux", "Toulouse", "Nantes", "Brest",
+            "Clermont-Ferrand", "La Rochelle", "Lille", "Metz", "Strasbourg", "Nancy", "Valence", "Avignon", "Montpellier",
+            "Nime", "Arles", "Nice", "Cannes");
+
+    public static String getARandomCity() {
+        return CITIES.get(new Random().nextInt(CITIES.size()));
+    }
+
+    @Transactional
+    public void init(@Observes StartupEvent ev, CustomerRepository customers, TransactionRepository transactions) {
+        Random random = new Random();
+        if (customers.count() == 0) {
+            var customer1 = new Customer();
+            customer1.name = "Clement Escofier";
```

Comment:
Missing one f in Escoffier :laughing: unless the name similarity is purely coincidental, of course!

Reply:
Man, that was a bad mistake, hope Clement did not see it :-)